nixos/modules/services/security/intune.nix

```diff
@@ -0,0 +1,51 @@
+{ config
+, pkgs
+, lib
+, ...
+}:
+with lib; let
+  cfg = config.services.intune;
+in
+{
+  options.services.intune = {
+    enable = mkEnableOption (lib.mdDoc "Microsoft Intune");
+  };
+
+
+  config = mkIf cfg.enable {
+    users.users.microsoft-identity-broker = {
+      group = "microsoft-identity-broker";
+      isSystemUser = true;
+    };
+
+    users.groups.microsoft-identity-broker = { };
+    environment.systemPackages = [ pkgs.microsoft-identity-broker pkgs.intune-portal ];
+    systemd.packages = [ pkgs.microsoft-identity-broker ];
+    systemd.services.microsoft-identity-device-broker.enable = true;
```

Comment:
If the service has a wantedBy we don't need this

Reply:
You're right, somehow got it into my head this was needed at some point. but the services are started from the dbus interfaces so not necessary :)